src/renderer/components/wallet/AssetDetails.tsx

```diff
@@ -29,6 +29,7 @@ type Props = {
   asset: O.Option<Asset>
   address: O.Option<Address>
   explorerUrl?: O.Option<string>
+  transferFees: TransferFeesRD
```

Comment:
im not sure but it seems `transferFees` is not used at this component now

Reply:
Yep, the logic by using `transferFees` will implemented with next PR.